indico/modules/events/editing/controllers/backend.py

```diff
@@ -250,3 +259,27 @@ def _process_PATCH(self, text, internal):
     def _process_DELETE(self):
         delete_revision_comment(self.comment)
         return '', 204
+
+
+class RHRevisionFilesExportZIP(RHContributionEditableRevisionBase):
+    def _check_revision_access(self):
+        return self._user_is_authorized_submitter() or self._user_is_authorized_editor()
+
+    def _process(self):
+        if not self.revision.files:
+            raise NotFound
+
+        temp_file = NamedTemporaryFile(suffix='indico.tmp', dir=config.TEMP_DIR)
```

Comment:
What about using a BytesIO instead of an actual file?

Reply:
Hm, yeah, good suggestion.